### ISSUE TITLE
Missing accessibility documentation

### ISSUE THREAD

**Issue by ClemMakesApps** [apache/incubator-echarts#9823]
I noticed that the echarts documentation website mentions 

> Rendering ability for ten-million-level data
> Supports both SVG and Canvas rendering
> Seperates data and style configuring
> Accessibility supports
> Wechart and Powerpoint supports

but I could not find any documentation on accessibility. Am I missing something or are we missing documentation?

As seen in https://ecomfe.github.io/echarts-doc/public/en/index.html

**Comment by Ovilia** [comment]
The tutorials are not all translated into English, and the related one for accessibility is missing for English. I will fix this later.
For the time being, you may refer to [aria in option document](https://ecomfe.github.io/echarts-doc/public/en/option.html#aria).
